andela/ah-bird-box
# #163388238 upgrade django from version 1.10 to 2.1

#### What does this PR do?
Have an upgrade for the application from Django 1.10 to Django 2.1
#### Description of Task to be completed?
1. Upgrade Django `pip install --upgrade django`
2. Change url to path on the url files
3. Include app_name on the url file in the authentication folder
4. Comment out Default Authentication classes on settings.py
#### How should this be manually tested?
Clone the repo and cd into it 

. checkout to the branch `git checkout ch-upgrade-django-version-163388238`

. `pip freeze` to view the dependancies

.  Django should read version 2.1

. `python manage.py runserver` to ensure server is running

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#163388238](https://www.pivotaltracker.com/n/projects/2240063)
#### Screenshots (if appropriate)
None
